modules/qna/src/backend/storage.ts

```diff
@@ -139,29 +150,103 @@ export default class Storage {
-    const qnaFilesPerTopic = await this.ghost.directoryListing(ROOT_FOLDER, 'qna.intents.json')
+    const qnaFilesPerTopic = await this.ghost.directoryListing(FLOW_FOLDER, 'qna.intents.json')
     const qnaPerTopic = {}
     for (const qnaFile of qnaFilesPerTopic) {
-      qnaPerTopic[qnaFile.split('/')[0]] = (await this.ghost.readFileAsObject<Intent[]>(ROOT_FOLDER, qnaFile)).length
+      qnaPerTopic[qnaFile.split('/')[0]] = (await this.ghost.readFileAsObject<Intent[]>(FLOW_FOLDER, qnaFile)).length
     }
     return qnaPerTopic
   }
 
   async deleteSingleItem(topicName: string, itemId: string) {
     await this.ensureIntentsFileExists(topicName)
-    const intents = await this.ghost.readFileAsObject<Intent[]>(ROOT_FOLDER, toQnaFile(topicName))
+    const intents = await this.ghost.readFileAsObject<Intent[]>(FLOW_FOLDER, toQnaFile(topicName))
     const newIntents = intents.filter(x => x.name !== itemId)
-    await this.ghost.upsertFile(ROOT_FOLDER, toQnaFile(topicName), serialize(newIntents))
+    await this.ghost.upsertFile(FLOW_FOLDER, toQnaFile(topicName), serialize(newIntents))
   }
 
-  async count(topicName: string) {
-    const questions = await this.fetchItems(topicName)
-    return questions.length
+  async exportPerTopic(topicName: string) {
+    const tmpDir = tmp.dirSync({ unsafeCleanup: true })
+    const zipName = path.join(tmpDir.name, `${topicName}.tar.gz`)
+    await mkdirp.sync(path.join(tmpDir.name, MEDIA_FOLDER))
+
+    const jsonQnaForBot = await this.ghost.readFileAsObject<Intent[]>(FLOW_FOLDER, toQnaFile(topicName))
+    const jsonQna = jsonQnaForBot.map(i => {
+      const newContent = [...i.metadata.contentAnswers].map(c => removeBotPrefix(c))
+      const newMetadata = { ...i.metadata, contentAnswers: newContent }
+      return { ...i, metadata: newMetadata }
+    })
+    const contentToExport: sdk.FormData[] = _.flatMapDeep(jsonQna.map(i => i.metadata.contentAnswers))
+    const mediaToExport = _.flatMapDeep(contentToExport.map(c => c.items ? c.items.map(i => i.image) : c.image))
+
+    await this.ghost.upsertFile(FLOW_FOLDER, 'exportable.qna.intents.json', serialize(jsonQna))
+
+    const filesToZip = [...mediaToExport, path.join(FLOW_FOLDER, 'exportable.qna.intents.json')]
+
+    for (const file of filesToZip) {
+      await mkdirp.sync(path.dirname(path.join(tmpDir.name, file)))
+      const buff = await this.ghost.readFileAsBuffer('/', file)
+      fse.writeFileSync(path.resolve(tmpDir.name, file), buff)
+    }
+
+    await tar.create(
+      {
+        cwd: tmpDir.name,
+        file: zipName,
+        portable: true,
+        gzip: true
+      },
+      filesToZip
+    )
+    const zipBuffer = await fse.readFile(zipName)
+    await this.ghost.deleteFile(FLOW_FOLDER, 'exportable.qna.intents.json')
+    tmp.setGracefulCleanup()
+    return zipBuffer
+  }
+
+  async importPerTopic(importArgs: ImportArgs) {
+    const { topicName, botId, zipFile, override, clean } = importArgs
+
+
+    if (!clean) {
```

Comment:
This seems to be the complete inverse logic. if(clean) then remove all questions right @allardy